### PR TITLE
Display gene panel failure info in flash message instead of terminal

### LINKED ISSUE #1755: Display more informative flash messages when a panel creation from file fails
A user was trying to create a new gene panel but couldn't (displayed message was  a generic "something went wrong in creating the panel"). With @moahaegglund we found out that it was because two genes in the panel file are missing in scout database:
- HLA-DRB3(4951)
- HLA-DRB4(4952)

It would be useful that instead of getting the generic error message the user could know what went wrong in the upload. For instance the scout log displays clearly: 
`WARNING hgnc_id 4951 is not in the gene database!`
`WARNING hgnc_id 4952 is not in the gene database!`

**Question, why are not these 2 genes in our database?**

### PR DESCRIPTION
**Pull request by northwestwitch**
fix #1755

The current situation is that whenever there is a gene-related error in gene panel creation, the user gets a general error message like this one:
 
image

And a a more specific error gets printed to log (terminal) instead, and the user doesn't directly know what is wrong with the panel unless he/she asks someone who has access to the log.

image

This PR changes the behavior to show a self-explanatory message to the user instead:

image

**How to test**:
1. On a local instance of Scout, modify the test panel with some non-existing genes (or for instance those from #1755) and try to create a new panel with that file.
1. See the difference between master and this branch

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by EM
- [x] tests executed by EM, CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
An easy review, anyone?

**Comment by elevu** [comment]
I can review!

File: scout/build/panel.py [hunk 1]
```diff
@@ -155,16 +155,18 @@ def build_panel(panel_info, adapter):
 
     gene_objs = []
     fail = False
```
Comment (northwestwitch) [inline scout/build/panel.py:157]: I'm going to actually remove this

**Comment by elevu** [comment]
Not stuff from your code but it prevents me from testing, as soon as I edit a gene in the table and try to upload it I get this error:
Screen Shot 2020-09-28 at 13 02 49
This is the gene I added: HLA-DRB3
Screen Shot 2020-09-28 at 12 56 21

**Comment by northwestwitch** [comment]
woa, which one are you editing the .txt or the csv?

**Comment by elevu** [comment]
.csv. Tried now with the .txt and it works!
Screen Shot 2020-09-28 at 13 37 06
Since it is not relevant to this PR, I can approve it and then open another issue for the bug.

**Review by elevu — APPROVED** [review]

**Comment by northwestwitch** [comment]
I see that the coverage went down: I'll try to add a test

**Pull request merged**